### PR TITLE
Add maintain TiDB Binlog documentations to facilitate the Binlog management in k8s

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What is changed, added or deleted? 
Add maintain TiDB Binlog documentations to facilitate the Binlog management in k8s

### What is the related PR or file link(s)? 
https://github.com/pingcap/tidb-operator/pull/744

### Which version does your change affect? 
`3.0`
I will align the `dev` version after this PR get approved.

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by aylei** [comment]
@dcalvin I've addressed the review comments, PTAL again

File: v3.0/tidb-in-kubernetes/reference/configuration/tidb-drainer.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+| `baseImage` | The base image of TiDB Binlog | `pingcap/tidb-binlog` |
+| `imagePullPolicy` | The pulling policy of the image | `IfNotPresent` |
+| `logLevel` | The log level of the drainer process | `info` |
+| `storageClassName` | `storageClass` used by the drainer. `storageClassName` refers to a type of storage provided by the Kubernetes cluster, which might map to a level of service quality, a backup policy, or to any policy determined by the cluster administrator. Detailed reference: [storage-classes](https://kubernetes.io/docs/concepts/storage/storage-classes) | `local-storage` |
+| `storage` | The storage limit of the drainer Pod. Note that you should set a larger size if `db-type` is set to `pb` | `10Gi` |
+| `disableDetect` |  Determines whether to disable casualty detection | `false` |
+| `initialCommitTs` |  Used to initialize a checkpoint if the drainer does not have one | `0` |
+| `config` | The configuration file passed to the drainer. Detailed reference: [drainer.toml](https://github.com/pingcap/tidb-binlog/blob/master/cmd/drainer/drainer.toml) | (see below) |
+| `resources` | The resource limits and requests of the drainer pod | `{}` |
```
Comment (dcalvin) [inline v3.0/tidb-in-kubernetes/reference/configuration/tidb-drainer.md]:
```suggestion
| `resources` | The resource limits and requests of the drainer Pod | `{}` |
```

File: v3.0/tidb-in-kubernetes/maintain/tidb-binlog.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    * Update an existing TiDB cluster to enable TiDB Binlog:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        helm upgrade <release-name> pingcap/tidb-cluster --version=<chart-version> -f <values-file>
+        ```
+
+## Deploy Multiple Drainers
```
Comment (dcalvin) [inline v3.0/tidb-in-kubernetes/maintain/tidb-binlog.md]:
```suggestion
## Deploy multiple drainers
```

**Comment by aylei** [comment]
@dcalvin Suggestion applied, PTAL
@tennix @weekface PTAL

File: dev/tidb-in-kubernetes/maintain/tidb-binlog.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm search tidb-drainer -l
+    ```
+
+2. Get the default `values.yaml` file to facilitate customization:
+
+    ```shell
+    helm inspect values pingcap/tidb-cluster --version=<chartVersion> > values.yaml
```
Comment (tennix) [inline dev/tidb-in-kubernetes/maintain/tidb-binlog.md]:
```suggestion
    helm inspect values pingcap/tidb-drainer --version=<chart-version> > values.yaml
```

File: dev/tidb-in-kubernetes/maintain/tidb-binlog.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    The `clusterName` and `clusterVersion` must match the desired source TiDB cluster.
+
+    For complete configuration details, refer to [TiDB Binlog Drainer Configurations in Kubernetes](tidb-in-kubernetes/reference/configuration/tidb-drainer.md).
+
+4. Deploy the drainer:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm deploy pingcap/tidb-drainer --name=<release-name> --namespace=<namespace> --version=<chart-version> -f values.yaml
```
Comment (tennix) [inline dev/tidb-in-kubernetes/maintain/tidb-binlog.md]:
```suggestion
    helm install pingcap/tidb-drainer --name=<release-name> --namespace=<namespace> --version=<chart-version> -f values.yaml
```

**Comment by aylei** [comment]
@tennix Suggestions has been applied to both `3.0` and `dev`, PTAL

**Pull request merged**